### PR TITLE
Type13 tweaks

### LINKED ISSUE #361: FR: add local receive timestamp to `--icmp-timestamp`
@moeller0 asked for the addition of a local receive timestamp value to the output from `--icmp-timestamp` in https://github.com/schweikert/fping/pull/353#issuecomment-2476739176. The existing `-D, --timestamp` option requires loop or count mode, and `--timestamp-format` does not provide the RFC 792 format of *milliseconds since midnight UTC*. Thus the output of `--icmp-timestamp` should be amended similarly to @moeller0's suggestion.

I too think that something like this would be useful for tracking *changes* in one way delay.

### PR DESCRIPTION
**Pull request by auerswal**
Some adjustments for ICMP Timestamp support. IMHO this does not need extra mention in `CHANGELOG.md`, maybe just add the pull request number to the line announcing ICMP Timestamp support.

Please see the individual commit messages for more information.

### CODE REVIEW

File: doc/fping.pod
Comment (gsnw-sebast) [inline doc/fping.pod:1]: With cap_net_raw the execution of fping is allowed and there is no socket error but ICMP_TIMESTAMP only works with root. The error then occurs `127.0.0.1: error while sending ping: Invalid argument`

Comment (gsnw-sebast) [reply doc/fping.pod:1]: Sorry, my mistake. My build script is broken at `setcap cap_net_raw,cap_net_admin+ep`

**Review by gsnw-sebast — APPROVED** [review]
No objections from my side. The changes are very good.

**Comment by moeller0** [comment]
Excellent! Looks great.

**Pull request merged**